doc/xsd/trc_db.xsd

```diff
@@ -40,6 +40,19 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:simpleType name="unknownExpStatus">
+        <xsd:annotation>
+            <xsd:documentation>
+                Enumeration of ways to handle unknown test/iteration.
```

Comment:
It says nothing about verdicts. It must be highlighted that it is PASSED without any verdicts.

Passed with verdicts must not be treated as expected. It must be UNKNOWN and require TRC entry to track it.

Reply:
In fact, it should be passed unexpectedly, if verdicts are met and unknown are OK.

Reply:
So it's:

- doc update
- and code update cause I think right now it ignores the verdicts.

Reply:
Sorry, it looks like it is not fixed

Reply:
I mean doc update